static/skin/index.js

```diff
@@ -280,20 +280,6 @@
                     divTag.innerHTML = `No result. Would you like to <a href="/?lang=">reset filter</a>?`;
```

Comment:
Just noticed that dropping the `onclick` handler exposes a bug in the URL which doesn't account for the `--urlRootLocation` option of kiwix-serve. If you fix it in this commit, don't forget to update the `setNoResultsContent()` in the next commit too.

Reply:
Did this while editing the commit message and to update setNoResultsContent() there's a fixup commit